skills/src/main/java/cm/aptoide/skills/SkillsFragment.kt

```diff
@@ -127,4 +137,31 @@ class SkillsFragment : DaggerFragment() {
 
     return intent
   }
+
+  private fun handleWalletCreationIfNeeded(): Observable<String> {
+    return viewModel.handleWalletCreationIfNeeded()
+        .observeOn(AndroidSchedulers.mainThread())
+        .doOnNext {
+          if (it == "CREATING") {
+            showWalletCreationLoadingAnimation()
+          }
+        }
+        .filter { it != "CREATING" }
```

Comment:
Should use a constant instead of an hardcoded string.

Reply:
agreed, changed